src/api.js

```diff
@@ -1,3 +1,3 @@
 export const API_KEYS = {
-    nearby:""
+    nearby:"AIzaSyCwOU9k7lloNkCQ5kdjzCSoyzyYWqtd-9I"
```

Comment:
This is a nono.

Reply:
Yes, but me we were getting annoyed of having to put it back in, and me and Colten didn't really things its a huge deal considering barely anyone is probably going to be using nearby, so congrats if you scrape our key (we currently use at most 10,000 out of our 8000000+ queries per day).